Hestia/Hestia/AppDelegate.cs

```diff
@@ -17,11 +25,115 @@ public override UIWindow Window
             set;
         }
 
+        public static UIStoryboard mainStoryboard = UIStoryboard.FromName(strings.mainStoryBoard, null);
+        public static UIStoryboard devices2Storyboard = UIStoryboard.FromName(strings.devices2StoryBoard, null);
+
+        string defaultServername;
+        string defaultIP;
+        string defaultPort;
+        string defaultAuth0AccessToken;
+        string defaultAuth0IdentityToken;
+
+        public bool IsServerValid()
+        {
+            try
+            {
+                bool validIp = PingServer.Check(defaultIP, int.Parse(defaultPort));
+            }
+            catch (Exception exception)
+            {
+                Console.Write(exception.StackTrace);
+                return false;
+            }
+            return true;
+        }
+
+        public bool IsAuth0LoginValid()
+        {
+            //TODO possibly a backend method that checks if token is still valid
+            return true;
+        }
+
+        public void SetGlobalsToDefaultsLocalLogin()
+        {
+            Globals.ServerName = defaultServername;
+            Globals.IP = defaultIP;
+
+            Globals.Port = int.Parse(defaultPort);
+            ServerInteractor serverInteractor = new ServerInteractor(new NetworkHandler(Globals.IP, Globals.Port));
+            Globals.LocalServerinteractor = serverInteractor;
+        }
+
+        public void SetGlobalsToDefaultsGlobalLogin()
+        {
+            //Globals.Auth0Servers = GetServers from auth0 backend method
+        }
+
         public override bool FinishedLaunching(UIApplication application, NSDictionary launchOptions)
         {
             // Override point for customization after application launch.
             // If not required for your application you can safely delete this method
+            NSUserDefaults userDefaults = NSUserDefaults.StandardUserDefaults;
+
+            // For Debugging
+           // Globals.ResetUserDefaults();
+           // userDefaults.RemoveObject(strings.defaultsServerNameHestia);
+           // userDefaults.RemoveObject(strings.defaultsIpHestia);
+           // userDefaults.RemoveObject(strings.defaultsPortHestia);
+           // userDefaults.RemoveObject(strings.defaultsLocalHestia);
```

Comment:
Please do not have commented out code.

Reply:
I will still need this when making the Segues after you add the auth0 code.